src/main/java/com/example/java/test/junior/developer/controller/UserController.java

```diff
@@ -0,0 +1,50 @@
+package com.example.java.test.junior.developer.controller;
+
+import com.example.java.test.junior.developer.dto.LanguageDto;
+import com.example.java.test.junior.developer.dto.UserDto;
+import com.example.java.test.junior.developer.model.User;
+import com.example.java.test.junior.developer.service.LanguageService;
+import com.example.java.test.junior.developer.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/user")
+public class UserController {
+
+    private final UserService userService;
+
+    @PostMapping("/save")
```

Comment:
This is not correct from REST point of view. 
So from REST point of view we have a **resource** - user. 
Not we have semantics from HTTP method - 
POST to create, GET to get, PUT to update, and DELETE to delete.
So if you want to expose an endpoint to CREATE a user - it should be `POST /api/v1/users` - NOT `/api/v1/users/create` or something. Consider that you already have "create" in HTTP method POST.